### PR TITLE
[kaleidescape] Add movie end time and music title channels

### PR DESCRIPTION
**Pull request by mlobstein**
This PR adds the following channels to the binding:
- **Title End Time:** The date/time when the currently playing movie will end. 
- **Music Title:** The raw output from the MUSIC_TITLE api response for use in rules that require track, artist and album changes in one update.

### CODE REVIEW

File: bundles/org.openhab.binding.kaleidescape/src/main/resources/OH-INF/thing/channels.xml [hunk 1]
```diff
@@ -376,6 +380,17 @@
...
 	</channel-type>
 
+	<channel-type id="endtime">
+		<item-type>DateTime</item-type>
+		<label>Title End Time</label>
+		<description>The date/time when the currently playing movie will end</description>
+		<category>Time</category>
+		<tags>
+			<tag>Timestamp</tag>
+		</tags>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.kaleidescape/src/main/resources/OH-INF/thing/channels.xml:390]: You can't set only a property, you have to combine it with a Point property.

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.kaleidescape/src/main/resources/OH-INF/thing/channels.xml (hunk 1 above)
Comment (mlobstein) [reply bundles/org.openhab.binding.kaleidescape/src/main/resources/OH-INF/thing/channels.xml]: Thanks. There are many examples of this without Point in the shelly binding.

Comment (lolodomo) [reply bundles/org.openhab.binding.kaleidescape/src/main/resources/OH-INF/thing/channels.xml]: It was not defined as expected in the Shelly binding !

Rather use a more specific value. I hesitate between "Status" and "Measurement".

**Review by lolodomo — APPROVED** [review]
LGTM, thank you

**Pull request merged**